### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #871 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#427]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 9.01 KB...]
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = null, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManager.lambda$put$1(UserProfileManager.java:131)

	at smartcity.accessibility.database.UserProfileManagerTest.testPut(UserProfileManagerTest.java:93)

Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.576 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  UserProfileManagerTest.testPut:93 
Argument(s) are different! Wanted:
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = org.parse4j.ParseFile@14ced6a2, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManagerTest.testPut(UserProfileManagerTest.java:93)
Actual invocation has different arguments:
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = null, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManager.lambda$put$1(UserProfileManager.java:131)


Tests run: 77, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:22 min
[INFO] Finished at: 2017-06-10T20:47:59+00:00
[INFO] Final Memory: 34M/213M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] 9534c7600b1cdfe5fc55a0836716f19805e9c07c - change to png and resize profile image #424

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/871/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 9.01 KB...]
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = org.parse4j.ParseFile@631e20b0, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManager.lambda$put$1(UserProfileManager.java:131)

	at smartcity.accessibility.database.UserProfileManagerTest.testPut(UserProfileManagerTest.java:93)

Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.536 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  UserProfileManagerTest.testPut:93 
Argument(s) are different! Wanted:
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = org.parse4j.ParseFile@e33a609, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManagerTest.testPut(UserProfileManagerTest.java:93)
Actual invocation has different arguments:
database.put(
    "UserProfile",
    {"numOfReviews" = 5, "rating" = 25, "profileImg" = org.parse4j.ParseFile@631e20b0, "username" = "alexaxa"}
);
-> at smartcity.accessibility.database.UserProfileManager.lambda$put$1(UserProfileManager.java:131)


Tests run: 77, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:14 min
[INFO] Finished at: 2017-06-10T20:54:29+00:00
[INFO] Final Memory: 34M/220M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] 9534c7600b1cdfe5fc55a0836716f19805e9c07c - change to png and resize profile image #424

- [yaelAmitay] f57f78c7d1bdcc1739093f20486b896fa836a6a0 - fix request parameters and return default img for now #424



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/872/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
